ale/formatters/__init__.py

```diff
@@ -1 +1 @@
-from . import usgscsm_formatter
+from . import usgscsm_formatter, isis_formatter
```

Comment:
Is this necessary? I'm not familiar enough with `__init__.py` files to know.

Reply:
In order to get these into the formatters namespace.